config/manager.go

```diff
@@ -196,24 +191,6 @@ func (f *FilesystemConfigManager) load(c *Config) error {
 		return err
 	}
 
-	if c.FirewallMark == 0 {
-		// without this users need to reset their configs
-		c.FirewallMark = defaultFWMarkValue
```

Comment:
Is this being handled in other place?

Reply:
The logic changed a bit. Also all this existed also into `newConfig()`, so it was just duplicated code.
Now the `c` is always created with the initial values `*c = *newConfig(f.machineIDGetter)` and then they are just replaced by `json.Unmarshal(decrypted, c)`. In this way when new settings are added can be automatically set to the default value from `newConfig`.